src/benchmarks/micro/MicroBenchmarks.csproj

```diff
@@ -121,6 +121,9 @@
   </ItemGroup>
 
   <ItemGroup>
+	<!-- Temporarily disable tests here -->
```

Comment:
Why does the indentation seem different?

Reply:
Apparently I don't have my Visual Studio setup with the Tabs vs Space properly.